packages/react-components/src/organisms/ResultList.tsx

```diff
@@ -53,23 +53,42 @@ const ResultList: React.FC<ResultListProps> = ({
 }) => (
   <article>
     <header css={headerStyles}>
-      <Paragraph primary>
-        <strong>
-          {numberOfItems} result{numberOfItems === 1 || 's'} found
-        </strong>
-      </Paragraph>
+      {numberOfItems > 0 && (
+        <Paragraph primary>
+          <strong>
+            {numberOfItems} result{numberOfItems === 1 || 's'} found
+          </strong>
+        </Paragraph>
+      )}
       {cardViewHref && listViewHref && (
-        <ListControls
-          isListView={isListView}
-          cardViewHref={cardViewHref}
-          listViewHref={listViewHref}
-        />
+        <div css={{ justifySelf: 'flex-end' }}>
```

Comment:
There's no flex anymore, so seems this div is dead?

Reply:
it right aligns the controls